nirc_ehr/test/src/org.labkey.test/tests.nirc_ehr/NIRC_EHRTest.java

```diff
@@ -810,26 +810,27 @@ public void testClinicalCasesWorkflow()
         setFormElement(Locator.textarea("openRemark"), "Clinical Case WorkFlow - Test");
         setFormElement(Locator.textarea("plan"), "Case plan");
         setFormElement(Locator.name("Id"), animalId);
-        _helper.getExt4FieldForFormSection("Clinical Case", "Open Date").setValue(LocalDateTime.now().minusDays(1).format(_dateFormat));
+        _helper.getExt4FieldForFormSection("Clinical Case", "Open Date").setValue(LocalDateTime.now().minusDays(2).format(_dateFormat));
```

Comment:
Looks like this is having a downstream affect on another test. When a clinical case is opened, daily observations start on that animal. I believe that is what is causing the row count to be off for testBehavioralCases. That test should probably be filtering on Observations before counting the rows.

What are we changing these dates for?

Reply:
@labkey-martyp I was seeing a failure with this test that was basically and end date wasn't after a before date (sorry I don't remember any more details).
I cannot remember where I was seeing that if it was on TeamCity or locally. Locally I run with the server in a different time zone, so maybe that was the issue.

At any rate I've backed out the changes to the DateTime values and have only left the simpler changes.

Let me know if you are ok with the those changes.